wallet/udb/addressmanager_test.go

```diff
@@ -1033,3 +1019,14 @@ func TestManager(t *testing.T) {
 	}
 	testManagerAPI(tc)
 }
+
+func TestMain(m *testing.M){
+	err := createEmptyDB()
+	defer os.RemoveAll(filepath.Dir(dbPath))
+
+	if err != nil {
+		fmt.Printf("Unable to create empty test db: %v", err)
+	}
+
+	m.Run()
```

Comment:
The return code should be used by os.Exit, otherwise failed tests wouldn't exit with a non-zero exit code.  Be careful that cleanup runs before os.Exit though.